chia/full_node/full_node.py

```diff
@@ -2133,75 +2133,67 @@ async def _can_accept_compact_proof(
             self.log.info(f"Duplicate compact proof. Height: {height}. Header hash: {header_hash}.")
         return is_new_proof
 
+    # returns True if we ended up replacing the proof, and False otherwise
     async def _replace_proof(
         self,
         vdf_info: VDFInfo,
         vdf_proof: VDFProof,
         height: uint32,
```

Comment:
can pass the header hash from the request into replace proof instead of using hight at all

Reply:
good point. I assumed the reason we made this lookup here was because the network message didn't include the hash, but it appears it does